package.json

```diff
@@ -1,98 +1,100 @@
-  },
-  "dependencies": {
-    "@ohdsi/atlascharts": "2.1.0",
-    "@ohdsi/ui-toolbox": "1.1.0",
-    "@ohdsi/visibilityjs": "2.0.2",
-    "ajv": "6.12.5",
-    "bootstrap": "3.4.1",
-    "bootstrap-select": "1.13.18",
-    "bowser": "2.11.0",
-    "clipboard": "2.0.11",
-    "colorbrewer": "1.5.3",
-    "crossfilter2": "1.5.4",
-    "d3": "4.13.0",
-    "d3-scale-chromatic": "1.5.0",
-    "d3-tip": "0.9.1",
-    "datatables": "1.10.18",
-    "datatables.net": "1.13.2",
-    "datatables.net-buttons": "2.3.4",
-    "datatables.net-select": "1.6.0",
-    "director": "1.2.8",
-    "eonasdan-bootstrap-datetimepicker": "4.17.49",
-    "express": "4.18.2",
-    "facets": "0.1.1",
-    "file-saver": "1.3.8",
-    "hash-it": "4.1.0",
-    "html2canvas": "1.4.1",
-    "jquery": "1.12.4",
-    "jquery-ui": "1.13.2",
-    "jquery-ui.autocomplete.scroll": "0.1.9",
-    "js-cookie": "2.2.0",
-    "papaparse": "5.3.2",
-    "jszip": "3.10.1",
-    "knockout": "3.5.1",
-    "knockout-mapping": "2.6.0",
-    "knockout-sortable": "1.2.2",
-    "leaflet": "1.9.3",
-    "less": "3.8.1",
-    "lodash": "4.17.21",
-    "lscache": "1.3.2",
-    "lz-string": "1.4.4",
-    "moment": "2.29.4",
-    "numeral": "2.0.6",
-    "ouanalyse-datatables.net-buttons-html5": "1.5.3",
-    "prismjs": "1.29.0",
-    "qs": "6.11.0",
-    "querystring": "0.2.1",
-    "svgsaver": "0.9.0",
-    "urijs": "1.19.11",
-    "venn.js": "0.2.20",
-    "xss": "1.0.14"
-  }
-}
+{
+  "name": "atlas",
+  "version": "2.15.0-DEV",
+  "description": "is an open source software tool for researchers to conduct scientific analyses on standardized observational data converted to the OMOP Common Data Model V5",
+  "main": "js/main.js",
+  "scripts": {
+    "prep": "npm i && npm run clean && npm run genversion",
+    "clean": "rimraf ./js/assets/bundle && rimraf ./js/assets/fonts && rimraf ./js/assets/images",
+    "build": "npm run prep && npm run build:dev",
+    "build:dev": "node build/optimize.js && npm run compress",
+    "build:docker": "npm run clean && npm run genversion && npm run build:dev && npm prune --production",
+    "compress": "terser ./js/assets/bundle/bundle.js -o ./js/assets/bundle/bundle.js -c --source-map",
+    "test": "jest",
+    "genversion": "genversion -s js/version/version.js && r_js -convert js/version js/ && rimraf js/version"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/OHDSI/Atlas.git"
+  },
+  "author": "Chris Knoll",
+  "contributors": [
+    "Alexander Saltykov <a_salt@live.ru>",
+    "Sigfried Gold",
+    "Frank DeFalco",
+    "Anthony Sena",
+    "Pavel Grafkin",
+    "Vitaly Koulakov",
+    "Jungmi Han",
+    "Mark Velez",
+    "Anton Gackovka"
+  ],
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/OHDSI/Atlas/issues"
+  },
+  "homepage": "https://github.com/OHDSI/Atlas#readme",
+  "devDependencies": {
+    "@babel/core": "7.24.7",
+    "@babel/plugin-transform-object-rest-spread": "7.24.7",
```

Comment:
This plugin is replaced with the new one as it's already deprecated.

Refs: 
+ Before - https://www.npmjs.com/package/@babel/plugin-proposal-object-rest-spread
+ After - https://www.npmjs.com/package/@babel/plugin-transform-object-rest-spread